### DISCUSSION TITLE
Week 8 Group Survey

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21S-DH151#66] [Assignments]
1. Group name
2. Group members
2. In general, how is the group doing?
2. What is the status of your code development?
3. What (if any) specific problems do you need the most help with?
3. What additional features are you thinking of adding to your project?
4. What other concerns do you have?

**Comment by MaryJaneHowland** [comment]
1. Happy Tweet

2. Ivan Ma, MJ Howland, Josephine Meyer-Hogan

3. The group is still struggling to finalize our second twitter dataset, which is stressing us out. Configuring the permissions and authentication to allow the right number of twitter data to come in has been a struggle. Twitter is not an application meant for geolocation, and this feature is optional for most users and tweets. It is very difficult to grab the right kind of data that we would need. Many of the geolocation features are premium as well on the Twitter API, so we have to find different workarounds. Basically, it is just additional work to hopefully get to the same result because of the lack of resources on it. Things should be ironed out by the end of this week. 

4. We are in the process of finalizing current pages for our site and drafting/adding other ones. Our about page is pretty much done, and we need to create a home page (MJ is working on), add to the facts and figures page, and potentially also create a conclusion/final thoughts page. We need to also finalize our maps and actually see if we can map twitter data (once we have it).

5. Finding data and using twarc. Possible accesses to premium features in Twitter API like has:geo and place:location would be nice. These are different fields that would help filter the search based on the API call. Trying to compile the data in a meaningful way has been a struggle. Our current plan is to try and collect tweets from each of the cities ranked in the happiness data so it can be a direct comparison between both data sets. The geolocation of the tweets has been a struggle so we are trying to search by the coordinates we are looking for. 

6. We want to add a level of user interactivity where a user will input their information including location info and happiness info, and that will show up on our maps.

7. We are just mostly concerned about timing. We want to make sure we get everything accomplished by finals.

**Comment by hoz-map** [comment]
1. Rethink Recycle

2. Alice, Hozumi, Emily

3. Good! We are on similar pages in terms of where we want our project to go and have been keeping each other updated on various aspects of the project.

4. Because our project is a single-scroll page, we are working on putting all the code together so that one index.html file can access the different maps.

5. Main issues so far were figuring out how to hide layers with a sidebar button, which Alex helped us out with! We are also currently running into some issues with getting more than one map to display on the page. After meeting with Yoh, the next step will be to go through our code to make sure that all variables and functions are distinctly named.

6. Looking ahead, we may consider potentially including a time component in our maps to help show how plastic waste management practices have changed throughout the years. This can also highlight how policy changes, such as China’s ban on importing plastic waste, have been felt globally. A lot of articles suggest that plastic wastes can harm our health, as well as the local environment. Thus, some other data to include would be the maps of the prevalence of certain diseases and polluted areas to visualize the effect of plastic wastes on the environment and health. Hopefully, those visualizations would help develop our narrative. In terms of the narrative, we are looking to incorporate a heavier write-up component within our project. This will help connect our narrative and explain how various interact. We also plan to supplement any research we have with additional, more humanistic resources -- currently, we are also considering looking at more intimate/personal narratives that reflect the impact of plastic waste management/mismanagement.

7. We currently don't have any major concerns.

**Comment by ryanorlando** [comment]
1. Team Data

2. Ryan Orlando, Ainsleigh Douglas, Sebastien Dills

3. Overall, our group is doing fairly well, but we still have a lot to do. One of our biggest goal for the next two weeks is to take our project and add features that set it apart from the basic requirements of the assignment. Another of our biggest struggles is the process of connecting our project focus to the data that we were able to find. All along we knew that our project would be more narrative and research-based, but we want to make sure that our maps and graphics add value to our presentation. This is an important issue and we want to be sure to do it justice.

4. Coding has probably been our biggest struggle so far. Most of our time has been spent on making the code actually work and meet the requirements, which has limited us in our creation of additional features. Our midterm project code was generally very clunky and the design elements were not up to our standards, but we are in the process of refining both our code and the visual appearance of our site.

5. One of the biggest things we need help with is finding the best way to display our data. We have already come to office hours to discuss possible features to add, and we liked a lot of the ideas we came up with. The big challenge will be figuring out how to code and incorporate these elements, so we may be scheduling more meetings to address that.

6. The most important additional feature we are planning to add is our emissions clock. We have already discussed the code and should have it added to the site soon. We also are considering adding some charts and tables to make our data more relevant and add to our project value. 

7. Our main concern is just timing and being able to generate the code to produce a site that we can be proud of. We have become very passionate about our topic, and we want to create something that reflects our knowledge and caring.

**Comment by alekhyavittalam** [comment]
1. State-of-the-Climate

2. Natasha Grace, Alekhya Vittalam, Zona Liao

3. At the moment, our group has been updating the overall aesthetic of the website by working in css to make the pages appear more clean with a clear homepage and toolbar settings for each of the pages. We are making progress on our web mapping by updating the code and continuing to stick to our original weekly plan to get everything done by the end of the quarter.

4. In terms of the status for our code development, we have been keeping up with the labs and adding elements we learn in class into our climate change web mapping. We just added the choropleth layer to our code and from there we want to continue fixing the colors and look of the map. ur code could do with some fine-tuning and maybe inputing the data/some new additions we're still figuring out how to add. It's missing a few things we feel could add a lot more to our site. For example, making the "Top 5" feature a function that's coded in instead of manually. We are currently working on incorporating a timeline in our maproom. the map will update the information based on the year that is selected on the bottom. Our code is not too complex since we are all beginner coders, but I think online sources and the class assignments has been helping us with our coding for our project map; we will continue to enhance the map and the features on it in our code.

5. I think our other main problem right now is the lack of user interactivity and the clarity of how the data is being conveyed. There's also a little bit of a disconnect in the data we have and the facts we got from other sources but we should be able to fix it. On clarity of data, we wish to use circle radiuses (for the markers) to fix the issue and review the datasets we are using.

6. I think some additional features we are thinking about adding to our project would be altering the legend to make it fit our map style while also updating it to the timeline slidebar that is currently selected by the users. We also thought it might be cool to make our map a black and white one or make it a variation of the original leaflet map to match the style of our webpage design more. We also wanted to add popups to our map to show more in depth research on our project.

7. Some concerns we have us that our choropleth layer uses data from the year 2016 whereas our other data sources use data from the year 2017. We also found some missing values in our data which we need to fix. We wanted to showcase the top countries with highest and lowest pollution levels and that is something we are trying to work on. Our main concern is the vizualisation of the data, we want it to deliver our message as intended but we're afraid it might not be meaningful enough. It is a little hard for us to see it from our perspective because of how closely we've been working with the data so we'll have other people review it to see how users can interact with it. There's a lot to improve on in terms of the interactions on the site and the clarity of our purpose.

**Comment by kmcinerny** [comment]
1. The Health Mappers

2. Kate & Frida

3. Our group is still working on coding and visualizations as well as the overall aesthetic and look of the website. There is constant communication about any issues and we usually work together on the coding. 

4. As of now, there are still some last minute features we need to work on. The data we are using is already in visual forms, we are just working on figuring out specific commands. We still need to figure out how to work a slider with our map and how to create popups that give more information about the mental health services in each zip code. In addition, we want to make sure users can toggle on and off layers of our choropleth map that show different features.

5. Mostly coding questions and debugging. We are still working on the literature review and “more resources” page. Also, we need approval for our page layout (will attend office hrs).

6. We want to add an automatic counter into the principal page.

7. What other concerns do you have? Just a little concerned about figuring out the coding, given the amount of time left and how long specific commands can take to figure out. We want to tell a straightforward, coherent story with our data, but can’t fully get to that point until we’ve finished some of our coding.

**Comment by LynnT2** [comment]
1. Fantastic Four
2. Labiba Alam, Adriana Romero, Lynn Tieu, Haiqi Zhou
3. Our group is doing good as of right now! We are making steady progress in our Final Project. We've successfully added a timeline, a choropleth map, a substantial About Me page, and established a consistent design theme. our team has kept up great communication with each other to make sure that we're all working together well, despite being remote.
4. The majority of our group has been able to develop HTML and CSS skills from the beginner level. The entire group has a grasp on creating a choropleth map, with some members with a firmer grasp on geojson than others. Haiqi is leading our group on the coding side, and is a great teacher 😄 . Our group is going to replace the current markers with a sidebar for our historical data, and include a color legend for our timeline. We have been able to successfully implement the features that we've coded on our site so far, and are confident that we can complete the site as we've envisioned.
5. We are still working on creating a solid storyline for our users to follow on our site, to best display the data that we have. Because the majority of our group is not very experienced in coding, a lot of our work has been put into improving our code and to make sure that it works.
6. We'd like to add a Resources page that goes beyond the data that we're displaying. It would provide links to organizations and databases that users can utilize if they're interested in further supporting the fight against or learning about the current attacks on Asian Americans. We also want to credit our data sources on a Data page. We're thinking of including a form on our site for people to fill out their own experiences as Asians in America, and to display the data from those submissions in real-time.
7. We have no concerns as of now!

**Comment by danielliuzy** [comment]
1. The Musical Maps
2. Daniel Liu, Kai Watanabe, Justis Guardia
3. Our group is doing really well! We are working on implementing the changes we discussed at the midterm. Together we are communicating on specific edits we are hoping to finish before the final.
4. Our code development is going well. The next big step we have is hopefully successfully utilizing and adding the twitter API to our current maps and data. 
5. We are not having any specific problems, but we anticipate some issues with the addition of a large amount of data.
6. As previously stated, our additional features include the Twitter API. We are also working on more seamlessly integrating the choropleth map into our site. We are editing some of the design elements as well and improving the site's accessibility, like making the map instructions more integrated into the main site.
7. Currently, we do not have any concerns and are pretty confident with our site and how it is coming along!

**Comment by michelle14lee** [comment]
1. Datart
2. Annmarie Cong, Eustina Kim, Michelle Lee
3. The group project has been going well. We've made progress each week according to our timeline and have begun to add additional features that we're interested in like a randomize function and gallery page. We communicate on a regular basis via Slack to make sure everyone on the team is updated about the progress and to troubleshoot issues together.
4. In terms of code development, we are about 70% finished. We have the "meat" of the website finished, but there are still several features that we're interested in implementing. We also plan to improve the aesthestic of the website and make it more user-friendly to various device types (phone, tablet, computer).
5. No problems currently, but we may ask for assistance/ideas on implementing a social media sharing feature of art pieces on Twitter or Instagram. We may also need help moving the Google Street View to the sidebar.
6. A few features we are planning/considering to add on the project are: Iteractive time slider, Art walk (routing between different art pieces), Social Media sharing (interactivity to share art pieces via Twitter or Instagram).
7. Our dataset is a bit outdated, so some of the art pieces don't exist anymore. We're debating between making the website a historical record of old art sites or to also incorporate an additional dataset with more recent art pieces. The trade-off is that the newer dataset doesn't have images of the actual site, which may create some inconsistency in our maproom.

**Comment by hanarama** [comment]
1. Environmental Racism in Carceral California
2. Julia Tanenbaum, Lisa Kahn, Hannah Rogers (nee Lien) 
3. We have a clear idea of the design features we want to include, but we are struggling with implementing some of them. On the whole, we are doing well, and though we may have to cut some features if we run out of time. 
4. Our code development seems to be progressing well, though we are running into some hiccups where we don't understand how to accomplish some of the things we want our site to do. As beginner coders, we are working hard to progress our knowledge beyond what is specifically addressed in the labs. 
5. We are really struggling with getting our search bar to function. We are also having issues in getting our markers to size in relation to population data. 
6. We are hoping to implement a search bar, and to create a more dynamic sidebar. We are working to put button controls for our choropleth layers into the sidebar. We also want to create a separate pages of additional information resources and featured stories. On the whole, we are looking to make our site more dynamic and digestible.  
7. We are unsure of how clear our message is. We want it to feel impactful and are looking for ways that we can get this across in the design. On a technical level, we are concerned with how much our site lags. We don't know how we would remedy this, or if we need to.

**Comment by derekwang99** [comment]
1. Ocean Pollution Posse
2. Eli Aranita, Julia Wood, Derek Wang
3. Our group is working together well, I think we all have the same expectations about how much work we should be putting into the group project and it aligns. I feel like we are also making considerable progress our putting a website together. We need to further develop the written narrative, but as for the code, we have several web maps for users to explore and draw connections between. All we need to do now is really weave it into a story through our design and written descriptions and context and history.
4. Our digital maps are completely coded, and we all have worked really hard in different aspects to bring this work. Our next steps in the code development entail UX coding using css.
5. I think we need the most help with developing our narrative of plastic pollution and our UX coding abilities so our design prototypes can come to life. 
6. We are still working on the interactive features so we will likely not have enough time for more features, but we would love to incorporate time-lapse somewhere
7.  We are a bit concerned with not having enough time seeing that it is already Week 8, but we feel that we will be able to get it all done.

**Comment by sebdills** [comment]
1)Team Data
2)Sebastien, Ryan, Ainsleigh 
3) think everything is going well the website works but is non-optimized and generally very slow. Otherwise things are going well. 
Our code works and the site is active.
4) I think having a more cohesive product, our data needs to fit our point better and service as more proper rhetoric. 
5) The cohesiveness of our data and visuals and adding the widget that shows energy consumption in real time. 
6) We are trying to add the widget I mentioned above. 
7) Not too many concerns.

**Comment by kaicolorado** [comment]
1. The cARTographers

2. Kai Colorado & Alex Haddad

3. Overall, the group is doing okay. We've been making solid progress and doing the best we can. The only issue is that we've spent a lot of time looking for the data we need, but it's been very difficult to find it. We still have a lot of data we can use and map out, but it doesn't seem to be enough for a strong narrative. We hope to supplement this narrative through the addition of more interactive and textual features that aid the user in understanding the focus of the project. By adding more features, the user is able to better understand the current state of policies in their area and the United States as a whole.

4. Good. Most of what's left is polishing the different aspects of the site and adding more information for users to give them context for what they're seeing. 

5. We may need help with getting our data to work for us and helping to build a narrative with what we have. Alternatively, if we were able to get the data we were looking for, this would be easier.

6. We were thinking about implementing a time slider that allows you to see how the data we're mapping has changed over time, but we're not sure if we'll have enough time to implement that. Currently, the arts education policies and assessment scores we're mapping are only for 2019. We are also planning on expanding the policies section of the site to provide a more complete narrative regarding the implementation of policies across states. Additionally, adding a table that would allow users to compare policies across states would increase the functionality of this policies page.

7. Our main concern is not having a clear narrative/message to show people that are using our map.
